src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/reports/ZipFilesSummaryRepository.java

```diff
@@ -13,30 +13,33 @@
-            + "  lastEventName.event AS status,\n"
-            + "  (CASE WHEN lastEventName.event = 'COMPLETED'\n"
-            + "        THEN lastEvent.eventDate ELSE null END\n"
-            + "  ) AS completedDate\n"
-            + "FROM (\n"
-            + "  SELECT container, zipfilename, MIN(createdat) AS createdDate FROM process_events \n"
-            + "  WHERE date(createdat) = :date AND event='ZIPFILE_PROCESSING_STARTED'\n"
-            + "  GROUP BY container, zipfilename\n"
-            + ") createdEvent \n" /* zipfile processing started date */
-            + "JOIN (\n"
-            + "  SELECT container, zipfilename, MAX(createdat) AS eventDate\n"
-            + "  FROM process_events \n"
-            + "  GROUP BY container, zipfilename\n"
-            + ") lastEvent \n" /* zipfile process completion date */
-            + "  ON createdEvent.container = lastEvent.container \n"
-            + "  AND createdEvent.zipfilename = lastEvent.zipfilename\n"
-            + "JOIN (\n"
-            + "  SELECT container, zipfilename, createdat, event\n"
-            + "  FROM process_events\n"
-            + ") lastEventName \n" /* latest event of the zipfile */
-            + "  ON lastEvent.container = lastEventName.container \n"
-            + "  AND lastEvent.zipfilename = lastEventName.zipfilename\n"
-            + "  AND lastEvent.eventDate = lastEventName.createdat\n"
+        value = "SELECT createdEvent.container, createdEvent.zipfilename, createdEvent.createdDate, "
+            + "  lastEventName.event AS lastEventStatus, envelope.status AS envelopeStatus, "
+            + "  (CASE WHEN lastEventName.event = 'COMPLETED' "
+            + "        THEN lastEvent.eventDate ELSE null END "
+            + "  ) AS completedDate "
+            + "FROM ( "
+            + "  SELECT container, zipfilename, MIN(createdat) AS createdDate FROM process_events "
+            + "  WHERE date(createdat) = :date AND event='ZIPFILE_PROCESSING_STARTED' "
+            + "  GROUP BY container, zipfilename "
+            + ") createdEvent " /* zipfile processing started date */
+            + "JOIN ( "
+            + "  SELECT container, zipfilename, MAX(createdat) AS eventDate "
+            + "  FROM process_events "
+            + "  GROUP BY container, zipfilename "
+            + ") lastEvent " /* zipfile process completion date */
+            + "  ON createdEvent.container = lastEvent.container "
+            + "  AND createdEvent.zipfilename = lastEvent.zipfilename "
+            + "JOIN ( "
+            + "  SELECT container, zipfilename, createdat, event "
+            + "  FROM process_events "
+            + ") lastEventName " /* latest event of the zipfile */
+            + "  ON lastEvent.container = lastEventName.container "
+            + "  AND lastEvent.zipfilename = lastEventName.zipfilename "
+            + "  AND lastEvent.eventDate = lastEventName.createdat "
+            + "LEFT JOIN envelopes AS envelope "
```

Comment:
This left join (and the related column in the select part) is the only change here. In other lines, I removed unnecessary `\n`s from the query, hoping the diff would be smarter about displaying those changes.